chopsticks/basilisk.yml

```diff
@@ -0,0 +1,27 @@
+endpoint:
```

Comment:
We don't have to do a sudo call on basilisk anymore, so we don't need this config, actually. However, I wanted to override the db path because it would be in the repository root.